hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMRecoverLeaseRequest.java

```diff
@@ -209,6 +213,13 @@ private RecoverLeaseResponse doWork(OzoneManager ozoneManager,
       throw new OMException("Open Key " + dbOpenFileKey + " not found in openKeyTable", KEY_NOT_FOUND);
     }
 
+    final long leaseSoftLimit = ozoneManager.getConfiguration()
```

Comment:
We should move this check before the OzoneConsts.LEASE_RECOVERY already exists check, otherwise, the second call of recoverLease will fail due to this soft limit check.

Reply:
Moved